azure-devops/azext_devops/dev/pipelines/pipeline_create.py

```diff
@@ -140,8 +144,10 @@ def pipeline_create(name, description=None, repository=None, branch=None, yml_pa
     created_definition = client.create_definition(definition=definition, project=project)
     logger.warning('Successfully created a pipeline with Name: %s, Id: %s.',
                    created_definition.name, created_definition.id)
-    return client.queue_build(
-        build=Build(definition=created_definition, source_branch=queue_branch), project=project)
+    if skip_first_run:
```

Comment:
In general do this command really need to trigger the pipeline created?
Customers can always trigger the created pipeline after creating pipeline using this command.
The current customer feedback looks like a good enough indication to me as to why commands should be atomic in nature.
tldr: I would prefer if this command never queues the created pipeline.

Reply:
I agree, user can always queue the pipeline later on by running separate command.

Reply:
Resolving since I have captured it together under one comment.